engine/src/test/java/org/camunda/bpm/engine/test/api/authorization/batch/DeleteHistoricProcessInstancesBatchAuthorizationTest.java

```diff
@@ -84,7 +84,7 @@ public void cleanBatch() {
   public static Collection<AuthorizationScenario[]> scenarios() {
     return AuthorizationTestRule.asParameters(
         AuthorizationScenarioWithCount.scenario()
-            .withCount(2L)
+            .withCount(1L)
```

Comment:
Why did this change?

Reply:
The historic instances are from different deployments. This count is only used in the test case `testWithTwoInvocationsProcessInstancesList`. Previously, both instances were processed in the same batch job execution in `DeleteHistoricProcessInstancesCmd`. In its `execute` method, both instances were checked for authorization. One of them would not pass and the whole batch job would fail, so no historic instance would be deleted (therefore still 2 instances would exist in the end).
With the new implementation, instances of different deployments will not be processed together in one batch job anymore. Therefore, 1 instance will be deleted now and one not.
I actually consider this the proper implementation for this case now.
Not sure if we need to document this but I would consider this a bug fix actually.

Reply:
I see! Thanks 👍